src/pages/api/auth/[...nextauth].jsx

```diff
@@ -262,15 +262,17 @@ const options = {
             session.data = token
             //loading adminsettings after creating users specific data as it does not belongs to specific user.
           //  if(session || !session.adminSettings)
-                session.adminSettings = await fetchUpdatedAdminSettings();
+          console.log('sess',session)
+              // session.adminSettings =await fetchUpdatedAdminSettings();
```

Comment:
Please remove all console statements and also test whether it works or not as i remembered it did not work.